lib/activerecord-clean-db-structure/clean_dump.rb

```diff
@@ -124,7 +124,7 @@ def run
         dump.gsub!(/^CREATE( UNIQUE)? INDEX \w+ ON .+\n+/, '')
         dump.gsub!(/^-- Name: \w+; Type: INDEX\n+/, '')
         indexes.each do |table, indexes_for_table|
-          dump.gsub!(/^(CREATE TABLE #{table}\b(:?[^;\n]*\n)+\);\n)/) { $1 + "\n" + indexes_for_table }
+          dump.gsub!(/^(CREATE TABLE #{table}\b(:?[^;\n]*\n)+\);*\n(?:.*);*)/) { $1 + "\n\n" + indexes_for_table }
```

Comment:
Test fails without this fix:

![image](https://github.com/user-attachments/assets/f6b8e0d8-2d37-4fce-abf3-1e59845d274b)